src/scripts/devhub.zig

```diff
@@ -34,6 +34,11 @@ pub fn main(shell: *Shell, gpa: std.mem.Allocator, cli_args: CliArgs) !void {
     const executable_size_bytes = (try shell.cwd.statFile("tigerbeetle")).size;
     try shell.project_root.deleteFile("tigerbeetle");
 
+    try shell.zig(
+        \\build scripts -- release --build --run-number=200 --sha={sha}
```

Comment:
Can we change this to 189 - the current release.zig asserts that it's being called with `0.15.4` as a target.